internal/testrunner/runners/system/servicedeployer/compose.go

```diff
@@ -104,6 +104,21 @@ func (r *DockerComposeServiceDeployer) SetUp(inCtxt ServiceContext) (DeployedSer
 	return &service, nil
 }
 
+// Signal sends a signal to the service.
+func (s *dockerComposeDeployedService) Signal(signal string) error {
+	p, err := compose.NewProject(s.project, s.ymlPath)
+	if err != nil {
+		return errors.Wrap(err, "could not create docker compose project for service")
+	}
+
+	opts := compose.CommandOptions{ExtraArgs: []string{"-s", signal}}
```

Comment:
Would you mind adding a sample package to the `test/packages` (a copy from integrations it's also fine), so we can check if this improvement works?

Reply:
I updated the multiinput test to exercise the feature. I added test log output to the PR description for reference.